src/wp-admin/includes/image.php

```diff
@@ -598,6 +598,9 @@ function _wp_make_subsizes( $new_sizes, $file, $image_meta, $attachment_id, $mim
 
 	if ( method_exists( $editor, 'make_subsize' ) ) {
 		foreach ( $new_sizes as $new_size_name => $new_size_data ) {
+			// Include size name in the data.
+			$new_size_data['name'] = $new_size_name;
```

Comment:
does this mean name will name be part of the image meta 'sizes' data?

Reply:
Not in the metadata, no. It's just gonna be passed to this method like that.